x-pack/plugins/cases/public/components/all_cases/utils.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parse } from 'query-string';
+
+import type { ParsedUrlQueryParams, UrlQueryParams } from '../../../common/ui/types';
+
+export const parseUrlQueryParams = (search: string): UrlQueryParams => {
+  const parsedUrlParams: ParsedUrlQueryParams = parse(search);
```

Comment:
I think we should use the native library [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to parse the params. What do you think?

Reply:
We discussed offline and decided to keep `parse`.